chia/full_node/full_node.py

```diff
@@ -323,7 +323,7 @@ async def _start(self) -> None:
         self._respond_transaction_semaphore = asyncio.Semaphore(200)
         # create the store (db) and full node instance
         # TODO: is this standardized and thus able to be handled by DBWrapper2?
-        async with create_connection(self.db_path) as db_connection:
+        async with manage_connection(self.db_path) as db_connection:
```

Comment:
Also shifting this so it ends up on the SQLite log as well.  https://github.com/Chia-Network/chia-blockchain/pull/13755

Reply:
is this the only call to `manage_connection`? there's no `log_path` or `name` ever passed in here.

Reply:
From my searching, yes, this is the only use outside of the wrapper module itself.  After #13755 it should end up specifying both `log_path` and `name`.